tsconfig.json

```diff
@@ -1,6 +1,15 @@
 {
-  "extends": "./node_modules/@polkadot/dev/config/tsconfig",
+  "exclude": [
+    "**/build/**/*",
+    "**/node_modules/**/*"
+  ],
   "compilerOptions": {
-    "types": []
+    "declaration": true,
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "jsx": "react",
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "es5"
```

Comment:
should we not target `esnext`?

Reply:
I'm not sure. If we do release publicly e.g. ui-components, then I really think it should be es5, I don't think esnext is that used yet.

Reply:
ok, in that case at least ES6 is pretty widely used, and I would want to keep things like the spread operator, and `a_${b}` syntax over `'a_'+'b'`, object destructuring, etc. Just referencing this https://medium.com/recraftrelic/es5-vs-es6-with-example-code-9901fa0136fc,